integration-tests/ci-visibility.spec.js

```diff
@@ -107,6 +108,10 @@ testFrameworks.forEach(({
   coverageMessage,
   type
 }) => {
+  // skip esm tests in node versions < 16
```

Comment:
```js
// to avoid this error: @istanbuljs/esm-loader-hook@0.2.0: The engine "node"
// is incompatible with this module. Expected version ">=16.12.0". Got "14.21.3"
```